pom.xml

```diff
@@ -55,21 +71,33 @@
   </modules>
 
   <properties>
+    <revision>HEAD-SNAPSHOT</revision>
+
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+
+    <!-- CI -->
+    <vertispan.j2cl.repo.url>https://repo.vertispan.com/j2cl/</vertispan.j2cl.repo.url>
+
+    <maven.compiler.plugin>3.8.1</maven.compiler.plugin>
     <maven.deploy.plugin>3.0.0-M1</maven.deploy.plugin>
+    <maven.flatten.plugin>1.2.2</maven.flatten.plugin>
     <maven.fmt.plugin>2.9</maven.fmt.plugin>
-    <maven.licence.plugin>3.0</maven.licence.plugin>
-  </properties>
+    <maven.gpg.plugin>1.6</maven.gpg.plugin>
+    <maven.gwt.plugin>1.0.0</maven.gwt.plugin>
+    <maven.javadoc.plugin>3.2.0</maven.javadoc.plugin>
+    <maven.j2cl.plugin>0.15-SNAPSHOT</maven.j2cl.plugin>
+    <maven.install.plugin>3.0.0-M1</maven.install.plugin>
```

Comment:
`maven.deploy.plugin` and `maven.install.plugin` could be removed

Reply:
removed `maven.install.plugin`-property. The `maven.deploy.plugin`-property can not be removed, cause it is needed.